lblod/frontend-lokaal-mandatenbeheer
# LMB-591 | Update replacement status of burgemeester

## Description

When a burgemeester of aangewezen burgmeester gets titelvoerend the mandataris status of the replacement should be set to waarnemend.

## How to test

Open a burgemeester and set to titelvoerend status. Select a replacer and the mandataris of the replacer should be a waarnemend status.
